### PR TITLE
ci: add DeleteLong to accommodate longer resource deletion

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by nebril** [comment]
test-gke

File: test/helpers/kubectl.go [hunk 1]
```diff
@@ -1465,6 +1465,13 @@ func (kub *Kubectl) Delete(filePath string) *CmdRes {
 		fmt.Sprintf("%s delete -f  %s", KubectlCmd, filePath))
 }
 
+// DeleteMiddle deletes the Kubernetes manifest at path filepath with longer timeout.
+func (kub *Kubectl) Delete(filePath string) *CmdRes {
```
Comment (aanm) [inline test/helpers/kubectl.go]: DeleteMiddle

**Review by aanm — CHANGES_REQUESTED** [review]

**Comment by nebril** [comment]
test-gke

**Comment by nebril** [comment]
test-4.9

**Comment by nebril** [comment]
test-gke

**Comment by nebril** [comment]
test-4.9

**Review by aanm — APPROVED** [review]

**Pull request merged**